cherrypy/_cpwsgi.py

```diff
@@ -54,33 +53,36 @@ class VirtualHost(object):
 
         cherrypy.tree.graft(vhost)
     """
-    default = None
-    """Required.
-
-    The default WSGI application.
-    """
 
+    default = None
     use_x_forwarded_host = True
```

Comment:
```suggestion
    use_x_forwarded_host = True
    """If True (the default), any "X-Forwarded-Host"
    request header will be used instead of the "Host" header. This
    is commonly added by HTTP servers (such as Apache) when proxying."""


```